index.html

```diff
@@ -9,225 +9,283 @@
       href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
       integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
       crossorigin="anonymous"
-      referrerpolicy="no-referrer"
-    />
+      referrerpolicy="no-referrer"/>
     <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="style2.css"/>
   </head>
   <body>
-    <section class="section-one">
-      <nav>
+
+<section class="header"> <!--wraps around section-one and section-two-->
+
+<section class="section-one">
+    <nav>
         <section class="logo">
           <a href="">My Logo</a>
         </section>
-        <aside>
-          <a href="">Portfolio</a>
-          <a href="">About</a>
-          <a href="">Contact</a>
-        </aside>
-        <section class="hamburger">
-          <img src="icon.jpg" alt="hamburger-icon" />
-        </section>
+
+          <aside>
+            <a href="">Portfolio</a>
+            <a href="">About</a>
+            <a href="">Contact</a>
+          </aside>
```

Comment:
**Optional:** As the previous reviewer didn't ask for that change, I kept it optional.

- [ ] Your Navbar is not matched with the Figma design. Please work on it.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![n1](https://user-images.githubusercontent.com/70483835/165926478-d1d324b7-942b-40d2-ae07-8bd3603b414f.png) | ![n2](https://user-images.githubusercontent.com/70483835/165926449-dcba12d6-3baf-484c-87e6-3a8266e6a876.png)